File: projects/order-creation-queries/core/src/main/java/com/robinfood/core/dtos/menu/MenuSuggestedPortionDataDTO.java
package com.robinfood.core.dtos.menu;

import lombok.AllArgsConstructor;
import lombok.Data;

@AllArgsConstructor
@Data
public class MenuSuggestedPortionDataDTO {

    private final Long id;
    private final String image;
    private final String name;
    private final Long parentId;
    private final String sku;
    private final Long unitId;
    private final Double unitNumber;
}

File: projects/posv2/core/src/main/kotlin/com/robinfood/core/dtos/menu/response/MenuSuggestedPortionDataDTO.kt
package com.robinfood.core.dtos.menu.response

data class MenuSuggestedPortionDataDTO(
        override var id: Long,
        override var image: String,
        override var name: String,
        override var parentId: Long,
        override var sku: String,
        var unitId: Long,
        var unitNumber: Double
) : MenuPortionDTO(id, image, name, parentId, sku)